lib/licensee/projects/uri_project.rb

```diff
@@ -0,0 +1,25 @@
+# URI-based project
+#
+# Analyzes a given URI for license information
+require 'net/http'
+require 'uri'
+
+module Licensee
+  class UriProject < Project
+    def initialize(path, **args)
+      @uri = URI(path)
```

Comment:
We may need to rescue from an invalid URI error here, in the event that I pass "http://foo"